emc-metalnx-services/src/main/java/com/emc/metalnx/services/irods/UserServiceImpl.java

```diff
@@ -440,9 +440,9 @@ public boolean updateOwnership(DataGridUser user, Map<String, Boolean> addCollec
 	public List<String> listUserTypes() {
 		List<String> userTypes = new ArrayList<String>();
 
+		userTypes.add(UserTypeEnum.RODS_USER.getTextValue());
 		userTypes.add(UserTypeEnum.RODS_ADMIN.getTextValue());
 		userTypes.add(UserTypeEnum.GROUP_ADMIN.getTextValue());
```

Comment:
Should `RODS_GROUP` be listed here or are groups already filtered out by the code triggering this path?

Just wondering.

Reply:
This is used to get the user types when adding user.  I don't think that one should be listed here.

Reply:
I see now.

Agreed.